src/main/resources/simple-auth-plugin.properties

```diff
@@ -3,6 +3,12 @@
 # Default state of server wide security
 simpleauth.default.active=false
 
+# Default response for publish security
+simpleauth.default.publish.response=true
+
+# Default response for playback security
+simpleauth.default.playback.response=true
+
```

Comment:
The code doesn't seem to be reading these two parameters. Based on their names, PlaybackSecurity.java and PublishSecurity.java should read these to set `defaultResponse`

Reply:
@david179 that seems logical to me